src/transformers/models/opt/modeling_opt.py

```diff
@@ -1065,6 +1070,7 @@ def forward(
         else:
             if input_ids is not None:
                 sequence_lengths = torch.ne(input_ids, self.config.pad_token_id).sum(-1) - 1
+                sequence_lengths = sequence_lengths.to(logits.device)
```

Comment:
This was to fix an `accelerate` test that was failing. I did not saw any error on the daily CI but I prefer to fix it as a sanity check and just to be sure

Reply:
Ha got it! It is an issue that we face with pytorch==1.13! I will revert that commit and let's keep that in mind as many `accelerate` tests could fail for pt==1.13 then
Will revert this change

Reply:
Fixed in 887705d